conf/solr/8.11.1/schema.xml

```diff
@@ -647,6 +647,9 @@
     <copyField source="westLongitude" dest="_text_" maxChars="3000"/>
     <!-- SCHEMA-COPY-FIELDS::END -->
```

Comment:
I feel like we need some documentation. Maybe something in the Search API page?

Reply:
Put something short in. Feel free to modify/add.

Reply:
Looks like a good start! Thanks much.